app/src/components/intelligence/memory-workspace.css

```
@@ -0,0 +1,617 @@
+  box-sizing: border-box;
+}
+
+.memory-workspace-root {
+  background: var(--paper);
+  color: var(--ink);
+  border-radius: var(--r-room);
+  border: 1px solid var(--hairline);
+  box-shadow: var(--shadow-rest);
+  overflow: hidden;
+  font-family:
+    'Inter',
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)